core/pages/Checkout.js

```diff
@@ -177,7 +183,12 @@ export default {
     onAfterPersonalDetails (receivedData, validationResult) {
       this.personalDetails = receivedData
       this.validationResults.personalDetails = validationResult
-      this.activateSection('shipping')
+
+      if (this.$store.state.cart.cartIsVirtual === true) {
```

Comment:
one missed here -> isVirtualCart

Reply:
i've updated branch, so only this one change left:)

Reply:
Added getter changes in 06fbfcc

Reply:
Also updated the branch with current develop